webroot/results/admin/check_rounds.php

```diff
@@ -91,6 +91,43 @@ function checkRounds () {
     list( $nbPersons, $competitionId, $year, $month, $day, $eventId, $roundId, $roundCellName, $formatId, $isNotCombined ) = $roundRow;
     $event = "$competitionId|$eventId";
 
+    $subsequentRoundCount = 0;
```

Comment:
Oh, I actually meant to change some other code to use $subsequentRoundCount
as well, thanks for the reminder!

On Sun, May 1, 2016 at 4:53 PM, Luis J. Iáñez <notifications@github.com>
wrote:

> In webroot/results/admin/check_rounds.php
> <https://github.com/cubing/worldcubeassociation.org/pull/563#discussion_r61695563>
> :
>
> > @@ -91,6 +91,43 @@ function checkRounds () {
> >      list( $nbPersons, $competitionId, $year, $month, $day, $eventId, $roundId, $roundCellName, $formatId, $isNotCombined ) = $roundRow;
> >      $event = "$competitionId|$eventId";
> >
> > +    $subsequentRoundCount = 0;
>
> Consider moving the calculation of $subsequentRoundCount into the next IF.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/563/files/d06fd081fd2602d0f426599616415e7b4c83a2d8#r61695563>
>